test/rekt/features/parallel/oidc_feature.go

```diff
@@ -14,49 +14,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package oidc
+package parallel
 
 import (
 	"context"
 	"strconv"
 
-	"knative.dev/eventing/test/rekt/features/featureflags"
-
-	cloudevents "github.com/cloudevents/sdk-go/v2"
+	v2 "github.com/cloudevents/sdk-go/v2"
 	"github.com/cloudevents/sdk-go/v2/test"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
why we name it v12?

Reply:
Probably come from the automated refactoring too. Good catch. I renamed it